docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -68,6 +69,8 @@ The `teleport-kube-agent` chart deploys the following Kubernetes resources:
 | `Role`                | The release name, suffixed by `-psp`                       | Enforces security requirements for pods deployed by `teleport-kube-agent`.                                                                                                                                                                                                                              | `podSecurityPolicy.enabled` is `true` and the Kubernetes cluster version is < 1.23.                               |
 | `RoleBinding`         | The release name, suffixed by `-psp`                       | Enforces security requirements for pods deployed by `teleport-kube-agent`.                                                                                                                                                                                                                              | `podSecurityPolicy.enabled` is `true` and the Kubernetes cluster version is < 1.23.                               |
 
+## Values reference
```

Comment:
Under this new H2, I would bump all intended subsections up a level, i.e., so H2s become H3s. We can then adjust `tocDepth` in the frontmatter to `3`.

Otherwise, this looks a bit odd in the table of contents.

Reply:
I'll leave it to a separate PR (together with new H2) to not mix content. This change is not directly related to the content of this PR.